np.sh

```diff
@@ -1,8 +1,18 @@
 #!/usr/bin/env sh
+
+if test -n "$(git status --porcelain)"; then
```

Comment:
Yeah, I want to warn on untracked files too, as this is for npm packages and I don't want anything not in git to end up in the package.